typings/artifacts.d.ts

```diff
@@ -4,8 +4,8 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import * as parseManifest from '../lighthouse-core/lib/manifest-parser.js';
-import * as _LanternSimulator from '../lighthouse-core/lib/dependency-graph/simulator/simulator.js';
+import parseManifest = require('../lighthouse-core/lib/manifest-parser.js');
```

Comment:
turns out we shouldn't be trying to `import * as` a commonjs module like this, so switched to the proprietary `import x = require('x.js')` (works only in typescript code)

Reply:
is this just for commonjs cases where you're not using a `declare module`? got a link to somewhere that has the offish word :)

Reply:
> the offish word :)

Probably [here in the handbook](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require), but there's a better explanation of the reasoning here from one of the typescript engineers:

> These are mostly equivalent, but `import *` has some restrictions that `import ... = require` doesn't. `import * as` creates an identifier that is a *module object*, emphasis on *object*. According to the ES6 spec, this object is never callable or `new`able - it only has properties.`

https://stackoverflow.com/a/35706271

In our case, it appears they also extend that to importing the type of the module's export, as our old way is broken with the more recent releases of typescript. In fact, he ends it with

> In some runtime+transpilation environments this might happen to work anyway, but it might break at any point in the future without warning, which will make you sad.

:)